recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jaws" %}
+{% set version = "0.2.6" %}
+{% set sha256 = "307d2e1343d8c16d4a0af836f2458339dfa57e09c0bf1559b567347f14ae3c53" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.11
```

Comment:
It doesn't seem that you are compiling against numpy, so you don't need this

Reply:
Also, the syntax would be `numpy 1.11.*` in the build section and `numpy >=1.11` in the run section.
So even if this compiles against `numpy` it is wrong.